bazel/external/system_libraries.BUILD

```diff
@@ -76,9 +94,10 @@ cc_library(
 cc_library(
     name = "libglog",
     srcs = glob(
-        ["usr/lib/*-linux-gnu/libglog.so.0"],
+        ["usr/lib/*-linux-gnu/libglog.so"],
```

Comment:
Why was this change necessary? Seems unrelated to the fluid and lfds changes.

Reply:
> Format filenames and linker flags for glog to be consistent.

This change is not really necessary, but this way the file and linker option specifications are consistent across the libraries in this file.

Reply:
discussed offline - checked that libglog.so is also in the arm package (/)